### ISSUE TITLE
Just for windows?

### ISSUE THREAD

**Issue by Ruttolo** [german-st/dialogflowv2#1]
Does it works only for windows?
I tried to install it on a docker on my Qnap NAS, this is the error:
 Unsupported platform for windows-build-tools@5.0.0: wanted {"os":"win32","arch":"any"} (current: {"os":"linux","arch":"x64"})   

kind regards,

**Comment by OctAg0nO** [comment]
same error on rpi 3

**Comment by danibjor** [comment]
Same here - running NodeRed in Docker.

```
27.11.2018 12:15:39 [warn] Installation of module node-red-contrib-dialogflowv2-api failed:
27.11.2018 12:15:39 [warn] ------------------------------------------
27.11.2018 12:15:39 [warn] npm ERR! Linux 4.4.0-131-generic
27.11.2018 12:15:39 npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "--save" "--save-prefix=~" "--production" "node-red-contrib-dialogflowv2-api@1.0.2"
27.11.2018 12:15:39 npm ERR! node v6.14.4
27.11.2018 12:15:39 npm ERR! npm  v3.10.10
27.11.2018 12:15:39 npm ERR! code EBADPLATFORM
27.11.2018 12:15:39
27.11.2018 12:15:39 npm ERR! notsup Unsupported platform for windows-build-tools@5.0.0: wanted {"os":"win32","arch":"any"} (current: {"os":"linux","arch":"x64"})
27.11.2018 12:15:39 npm ERR! notsup Valid OS:    win32
27.11.2018 12:15:39 npm ERR! notsup Valid Arch:  any
27.11.2018 12:15:39 npm ERR! notsup Actual OS:   linux
27.11.2018 12:15:39 npm ERR! notsup Actual Arch: x64
27.11.2018 12:15:39
27.11.2018 12:15:39 npm ERR! Please include the following file with any support request:
27.11.2018 12:15:39 npm ERR!     /data/npm-debug.log
27.11.2018 12:15:39
27.11.2018 12:15:39 [warn] ------------------------------------------
```

**Comment by german-st** [comment]
Hi.

Apparently node-pre-gyp demanded a python and installed it. I will try to remove it from the modules and update the version. Thanks

**Comment by german-st** [comment]
Guys, @Ruttolo @OctAg0nO @danibjor 
I removed unnecessary dependencies. I tested the installation on Ubuntu and Windows after this. Everything went right. Please try again. Working version 1.0.4 or @latest
The command for install:
`npm i node-red-contrib-dialogflowv2-api`

Write the results, please. It's important for me. Thank you all.

**Comment by german-st** [comment]
Fixed in version 1.0.4
For install: `npm i node-red-contrib-dialogflowv2-api`
or
`npm i node-red-contrib-dialogflowv2-api@1.0.4`
